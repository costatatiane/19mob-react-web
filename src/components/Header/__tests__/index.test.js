import React from 'react';
import renderer from 'react-test-renderer';

import Header from '../index';

test('snapshot', () => {
    const element = renderer.create(<Header />)
    expect(element.toJSON()).toMatchSnapshot();
})