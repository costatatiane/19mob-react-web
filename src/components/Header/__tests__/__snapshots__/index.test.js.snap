// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <img
      className="logo"
      src="logo.png"
    />
  </div>
</header>
`;
